src/helpers/iboss-status.ts

```diff
@@ -1,9 +1,10 @@
 /**
- * checks if the CH_IBOSS_TRIAL variable is set in the environment as a truthy value
+ * checks if the CH_IBOSS_TRIAL variable is set in the environment as a boolean value
  *
  * @returns boolean
  */
 export const isIbossEnabled: () => boolean = () => {
     const env = process.env.CH_IBOSS_TRIAL?.toLowerCase();
-    return !!env && env === "true";
+    const invalidValues = new Set(["false", "no", "0"]);
+    return !!(env && !invalidValues.has(env));
```

Comment:
What if it was `blah` it would be true - is this OK? Wouldn't it be better to have `validValues`? easier to understand

Reply:
Yeah, so Tom suggested that we track only those invalidValues, anything else should be truthy.